integrations/opencost/main.py

```diff
@@ -1,11 +1,34 @@
+from enum import StrEnum
 from typing import Any
 
-from client import OpenCostClient
 from port_ocean.context.ocean import ocean
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+from client import OpenCostClient
+
+
+class ObjectKind(StrEnum):
+    COST = "cost"
+    CLOUDCOST = "cloudcost"
```

Comment:
what is the difference?

Reply:
The first deals with K8 costs, the second deals with costs per provider